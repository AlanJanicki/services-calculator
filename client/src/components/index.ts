export { Card } from './Card/Card';
export { Checkbox } from './Checkbox/Checkbox';
export { Select } from './Select/Select';
export { Spinner } from './Spinner/Spinner';
