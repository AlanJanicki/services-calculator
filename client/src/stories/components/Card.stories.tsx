import { Card } from '../../components';

export default {
  component: Card,
  tags: ['autodocs'],
  title: 'Card'
};

export const Default = {
  args: {
    header: 'Kalkulator us≈Çug'
  }
};
