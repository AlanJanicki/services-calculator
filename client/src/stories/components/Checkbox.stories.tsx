import { Checkbox } from '../../components';

export default {
  component: Checkbox,
  tags: ['autodocs'],
  title: 'Checkbox'
};

export const Default = {
  args: {
    name: 'Internet'
  }
};
