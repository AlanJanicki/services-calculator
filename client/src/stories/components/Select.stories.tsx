import { Select } from '../../components';
import priceList from '../../mocks/servicesPriceList.json';

export default {
  component: Select,
  tags: ['autodocs'],
  title: 'Select'
};

export const Default = {
  args: {
    label: 'Wybierz rok',
    options: priceList.years
  }
};
