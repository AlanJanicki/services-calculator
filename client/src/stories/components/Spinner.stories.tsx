import { Spinner } from '../../components';

export default {
  component: Spinner,
  tags: ['autodocs'],
  title: 'Spinner'
};

export const Default = {};
