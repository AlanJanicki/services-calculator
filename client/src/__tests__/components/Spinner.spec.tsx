import React from 'react';

import { Spinner } from '../../components';
import { renderWithSnapshot } from '../common';

describe('<Spinner/>', () => {
  it('matches snapshot', () => renderWithSnapshot(<Spinner />));
});
