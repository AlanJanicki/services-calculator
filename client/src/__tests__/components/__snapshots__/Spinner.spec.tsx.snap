// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/> matches snapshot 1`] = `
<DocumentFragment>
  .c0 {
  display: block;
  width: 50px;
  aspect-ratio: 1;
  border: 3px solid #0f7bff;
  border-radius: 50%;
  border-top-color: #fff;
  -webkit-animation: spin 1s ease-in-out infinite;
  animation: spin 1s ease-in-out infinite;
}

<p
    role="status"
  >
    <span
      class="c0"
      width="50"
    />
  </p>
</DocumentFragment>
`;
